src/webgpu/capability_info.ts

```diff
@@ -162,9 +166,11 @@ export const kSizedDepthStencilFormatInfo: {
     readonly bytesPerBlock: number;
     readonly blockWidth: 1;
     readonly blockHeight: 1;
+    dataType: TextureDataType;
+    componentType: GPUTextureComponentType;
   } & TextureFormatInfo;
 } = /* prettier-ignore */ {
-  'depth32float': { renderable:  true, color: false, depth:  true, stencil: false, storage: false, copySrc:  true, copyDst: false, bytesPerBlock:  4, blockWidth: 1, blockHeight: 1 },
+  'depth32float': { renderable:  true, color: false, depth:  true, stencil: false, storage: false, copySrc:  true, copyDst: false, bytesPerBlock:  4, blockWidth: 1, blockHeight: 1, dataType:  'float', componentType: 'float' },
```

Comment:
Mmmh can't this also be "depth-comparison"?  How are we going to handle formats that support multiple components types? (or is this just for sampling in a texture to read the value and nothing else?)

Reply:
I'm probably going to need to make this an array or we'll have to add per-aspect information at some point.. but for now it's just for the sampling part.

Reply:
Given I think we're changing 'depth-comparison' to 'depth-stencil' upstream, maybe we don't even need to support 'float' component type for depth formats?